coffee_github/src/repository.rs

```diff
@@ -255,6 +251,9 @@ impl Repository for Github {
         }
         // pull the changes from the repository
         let status = fast_forward(&self.url.path_string, &self.branch)?;
+        // update the git information
+        // (git HEAD and date of the last commit)
+        self.git_info().await?;
```

Comment:
I would like to see something like `let (commit, time) = get_repo_info!()`  and then so just call, `self.commit = commit`